geocoding/lib/geocoding.dart

```diff
@@ -15,13 +15,12 @@ export 'package:geocoding_platform_interface/geocoding_platform_interface.dart';
 /// When not supplied the currently active locale of the device will be used.
 /// The `localeIdentifier` should be formatted using the syntax:
 /// [languageCode]_[countryCode] (eg. en_US or nl_NL).
-Future<List<Location>> locationFromAddress(
+Future<List<Location>>? locationFromAddress(
```

Comment:
I don't think we should make the return value "nullable" as that would mean a breaking change. Second there should always be a platform implementation or otherwise an exception should be thrown as the platform is not supported or not configured correctly.
```suggestion
Future<List<Location>> locationFromAddress(
```